frontend/src/services/mock.ts

```diff
@@ -24,4 +28,35 @@ export const mockUser: User = {
 export const mockWallet: IWallet = {
     id: 40,
     balance: 100,
+}
+
+export const mockExpensesTransactions: IExpenseTransaction = {
+    transactions: [
+        {
+            amount: 30,
+            nameTo: "Starbucks",
```

Comment:
don't you need a percentage of each part?

Reply:
can i calculate percantage by using mongo or i should do it in separate method?

Reply:
You can do it in mongo, but if it's hard you can calculate it right after you get data from the db. But not on the client side